process/track/baseBlockTrack.go

```diff
@@ -89,6 +117,10 @@ func (bbt *baseBlockTrack) receivedMetaBlock(headerHandler data.HeaderHandler, m
 	bbt.blockProcessor.processReceivedHeader(metaBlock)
 }
 
+func (bbt *baseBlockTrack) checkBlockBasicValidity(_ data.HeaderHandler) error {
```

Comment:
why is this needed? does it need to implement any interface that requires this? otherwise I would remove it.

Reply:
function removed